### ISSUE TITLE
Compromise tags "there" incorrectly when followed by "is"

### ISSUE THREAD

**Issue by roschler** [spencermountain/compromise#1121]
Hello.  Given this sentence:
```js
const testSentence = 'There is a dog.';

  const aryTestCompromisedNlpTermObjs =
    doc.terms().json();
```

The `compromise` _nlp(testSentence)_ generates this output for the first element in the _terms().json()_ output:

``
‌{"text":"There","terms":[{"text":"There","pre":"","post":" ","tags":["There"],"normal":"there","index":[0,0],"id":"there|00100000P","chunk":"Noun","dirty":true}]}
```
Note, it may be a problem with "there" in general. I tested it with a few other sentences (e.g. - "There can never be peace"), and the same error occurred.
As you can see, it tagged "there" with the word "there" instead of the part of speech label "Adverb".

**Comment by spencermountain** [comment]
ha! yeah, at first i was sure this was a bug too, and then I remembered the [Existential There Tag](https://www.ling.upenn.edu/courses/Fall_2003/ling001/penn_treebank_pos.html) that appears in the penn treebank, and that we adopted a few years ago.

Happy to change things, if this is too awkward. From what I understand, it's a complex word!
cheers

**Comment by roschler** [comment]
Understood.  It might be better thought to make the tag something like `ExistentialThereTag` or something else orthogonal from the word.  The proof of the need for this being that even you, the author, has the same perception until you too remember that it was a valid tag in the Penn treebank.   In the meantime, I have added post-processing code for after the `terms().json()` call to force the part-of-speech for that word to `pronoun`.  I looked at all the cases for "there" and I am aware of the `adverb/adjective/existential/etc.` cases, but for my NLP parsing purposes those seem to be a bit "touchy feely" for strict parsing purposes.

Thank you for this package.  It fills the void for the need of something a lot more capable then the easily confused lower tier Node.js NLP packages, and having to resort to a heavyweight beast like the Standford NLP package with its steep learning curve and heavyweight processing needs.
